PostHog/PostHogStorage.swift

```diff
@@ -75,16 +95,30 @@ class PostHogStorage {
         case sessionReplay = "posthog.sessionReplay"
         case isIdentified = "posthog.isIdentified"
         case personProcessingEnabled = "posthog.enabledPersonProcessing"
+
+        var useFileStorage: Bool {
```

Comment:
Not sure i follow why we need this

Reply:
I think its common that the OS clears up caching whenever is needed, but I don't think its common that it clears the cache every time you exit the app, is it?
if that's not the case, I'd just use the cache always, so we don't have to have this extra logic around `useFileStorage`

Reply:
I also understand that using the `UserDefaults` would help persisting the distinct id whenever this happens, if it happens, so your call here